templates/overview/index.jsx

```diff
@@ -51,7 +46,6 @@ const OverviewTemplate = ({
       enrichmentSize={doiEnrichmentSize}
       dataProviderId={dataProviderId}
     />
-    <PlaceholderCard title="ORCiDs" value={5.8} />
```

Comment:
You can remove PlaceholderCard component too. If we need it we can revert it later.

Reply:
I would like to keep it at the moment. I am almost sure we will get a request to put it back or create another card.

PlaceholderCard will disappear when we move everything to Nivo charts because linter will point that recharts got removed.